integration_tests/e2e/induction/createInduction.cy.ts

```diff
@@ -311,7 +311,7 @@ context('Create an Induction', () => {
               "@.personalSkillsAndInterests.interests[1].interestType == 'DIGITAL' && " +
               '@.workOnRelease.affectAbilityToWork.size() == 1 && ' +
               "@.workOnRelease.affectAbilityToWork[0] == 'NONE' && " +
-              "@.workOnRelease.affectAbilityToWorkOther == '' && " +
+              '!@.workOnRelease.affectAbilityToWorkOther && ' +
```

Comment:
The "other" text fields are now only mapped if the corresponding answer is/includes "OTHER", hence this change